contracts/StakeManager.sol

```diff
@@ -222,9 +222,18 @@ contract StakeManager is Ownable {
         if (deltaTime < MIN_LOCKUP_PERIOD || deltaTime > MAX_LOCKUP_PERIOD) {
             revert StakeManager__InvalidLockTime();
         }
-        _mintInitialMP(account, _timeToIncrease, 0);
-        //update account storage
```

Comment:
`_mintInitalMP()` increases `account.initialMP` which I don't think we want when we `lock()`